game/static/game/css/landing.css

```
@@ -53,18 +70,29 @@ body {
 .logo-link {
     display: flex;
     align-items: center;
-    gap: 1rem;
+    gap: 0.75rem;
     text-decoration: none;
     color: var(--text-primary);
     font-family: 'Cinzel', serif;
```

Stylelint (17.11.0)

[error] Expected no quotes around "Cinzel" (font-family-name-quotes)

(font-family-name-quotes)